src/components/Calculator.js

```diff
@@ -40,20 +41,22 @@ const Calculator = () => {
     { id: 'equal', label: '=' },
   ];
   return (
-    <section className="main-container">
-      <Output value={displayValue} />
-
-      {buttons.map((btn) => (
-        <button
-          key={btn.id}
-          type="button"
-          className={btn.id}
-          onClick={() => handleBtnClick(btn.label)}
-        >
-          {btn.label}
-        </button>
-      ))}
-    </section>
+    <>
+      <section className="main-container">
+        <Output value={displayValue} />
+        {buttons.map((btn) => (
+          <button
+            key={btn.id}
+            type="button"
+            className={btn.id}
+            onClick={() => handleBtnClick(btn.label)}
+          >
+            {btn.label}
+          </button>
+        ))}
+      </section>
+      <Quotes />
```

Comment:
Nice work so far. However, instead of importing the `Qoute` component here, kindly import it into your main component (next to the Calculator component and not inside). Remember, this is part of the project requirements. :+1: